src/main/java/com/ericsson/ei/frontend/WebController.java

```diff
@@ -43,29 +43,21 @@
 
     @RequestMapping("/")
     public String greeting(Model model) {
+        model.addAttribute("frontendServiceUrl", getFrontendServiceUrl());  // inject in DOM for AJAX etc
         String eiffelDocumentationUrlLinks = String.format("%s", eiffelDocumentationUrls);
         model.addAttribute("eiffelDocumentationUrlLinks", eiffelDocumentationUrlLinks);  // inject in DOM for AJAX etc
         return "index";
     }
 
     @RequestMapping("/subscriptionpage.html")
     public String subscription(Model model) {
-        String httpMethod = "http";
-        if (backEndInformation.isHttps()) {
-            httpMethod = "https";
-        }
-        String frontendServiceUrl;
-        if (frontendContextPath != null && !frontendContextPath.isEmpty()) {
-            frontendServiceUrl = String.format("%s://%s:%d/%s", httpMethod, frontendServiceHost, frontendServicePort, frontendContextPath);
-        } else {
-            frontendServiceUrl = String.format("%s://%s:%d", httpMethod, frontendServiceHost, frontendServicePort);
-        }
-        model.addAttribute("frontendServiceUrl", frontendServiceUrl);  // inject in DOM for AJAX etc
+        model.addAttribute("frontendServiceUrl", getFrontendServiceUrl());  // inject in DOM for AJAX etc
         return "subscription";
     }
 
     @RequestMapping("/testRules.html")
     public String testRules(Model model) {
+        model.addAttribute("frontendServiceUrl", getFrontendServiceUrl());  // inject in DOM for AJAX etc
```

Comment:
Is it required to have frontendServiceUrl add to every page?

Reply:
If we want to have it in each ajax request then yes. in other case in js file we will have empty string when call frontendServiceUrl

Reply:
Ok,, just checking so we use the variable on all page.